bool arrived
uint16 start_navigation
